apps/daimo-mobile/src/logic/deeplink.ts

```diff
@@ -0,0 +1,29 @@
+import { getInitialURL } from "expo-linking";
+import { Platform } from "react-native";
+import NfcManager, { Ndef } from "react-native-nfc-manager";
+
+export async function getInitialURLOrTag() {
+  const url = await getInitialURL();
+  if (url) return url;
+
+  if (Platform.OS === "android") {
+    const tag = await NfcManager.getLaunchTagEvent();
+    if (tag && tag.ndefMessage.length > 0) {
+      console.log(`[DEEPLINK] got NFC tag ${JSON.stringify(tag)}`);
+      try {
+        const message = tag.ndefMessage[0];
+        const url = Ndef.uri.decodePayload(
+          message.payload as unknown as Uint8Array
```

Comment:
kinda ugly cast...  avoid if possible, otherwise OK

Reply:
found a way to avoid but will have to test new logic straight in release candidate build lol